pkg/sfu/buffer/buffer.go

```diff
@@ -500,12 +500,12 @@ func (b *Buffer) updateStreamState(p *rtp.Packet, arrivalTime int64) {
 	}
 }
 
-func (b *Buffer) processHeaderExtensions(p *rtp.Packet, arrivalTime int64) {
+func (b *Buffer) processHeaderExtensions(p *rtp.Packet, arrivalTime time.Time) {
 	// submit to TWCC even if it is a padding only packet. Clients use padding only packets as probes
 	// for bandwidth estimation
 	if b.twcc != nil && b.twccExt != 0 {
 		if ext := p.GetExtension(b.twccExt); ext != nil {
-			b.twcc.Push(binary.BigEndian.Uint16(ext[0:2]), arrivalTime, p.Marker)
+			b.twcc.Push(binary.BigEndian.Uint16(ext[0:2]), arrivalTime.UnixNano(), p.Marker)
```

Comment:
I would like to change this also to use `time.Time` so that we have monotonic clock in TWCC also. But, did not want to introduce that dependency in this PR. Will look at it later.